mtags/src/main/scala-2/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -328,7 +328,8 @@ class CompletionProvider(
         !isFileAmmoniteCompletion() &&
         completion.isCandidate(head) &&
         !head.sym.name.containsName(CURSOR) &&
-        isNotLocalForwardReference
+        isNotLocalForwardReference &&
+        !isAliasCompletion(head)
```

Comment:
@kpodsiad @tanishiking 
I updated PR description.

This line was added because [this test](https://github.com/scalameta/metals/blob/c2c6b7077f46b9d30e9be3806691e792919bfa92/tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala#L94-L107) and similar to it started failing.

Having the following source:
```
object Main {
   type Bar = List[Int]
   val x: Ba@@
}
```
compiler started returning completion: List(`Bar`, `List`) which is incorrect and might be filtered out by checking over a new field `aliasInfo` in Type and Scope Member

Reply:
`List` should be actually removed due to fuzzy matching, the compiler can return `List` , but it should not be used by Metals

Reply:
Ach I see, the compiler gives us the name `MT` instead of List, that seems like a bug really :thinking: 

I am looking if we can avoid adding new directory for Scala 2.13.9

Reply:
Alternative I found is:
```
def isAliasType = head.getClass().getMethods().find(_.getName() == "aliasInfo") match {
          case Some(aliasType) => aliasType.invoke(head) match {
            case Some(_) => true
            case _ => false
          }
          case _ => false

        }
```

though that uses reflection :/